### PR TITLE
sort files by globs then runtime

### PR DESCRIPTION
**Pull request by JonErikDSuero**
For https://github.com/nulogy/Gorgon/issues/72

### CODE REVIEW

File: spec/runtime_file_reader_spec.rb [hunk 1]
```diff
@@ -3,47 +3,75 @@
...
 
     it "should remove old files that are not in current files" do
       current_spec_files = ["new_a.rb", "old_a.rb", "old_c.rb"]
-      sorted_files = @runtime_file_reader.sorted_files(current_spec_files)
-      expect(sorted_files.first(2)).to eq(["old_a.rb", "old_c.rb"])
-      expect(sorted_files.last(1)).to eq(["new_a.rb"])
+      sorted_files_by_runtime = @runtime_file_reader.send(:sorted_files_by_runtime, current_spec_files)
+      expect(sorted_files_by_runtime.first(2)).to eq(["old_a.rb", "old_c.rb"])
+      expect(sorted_files_by_runtime.last(1)).to eq(["new_a.rb"])
     end
```
Comment (JonErikDSuero) [inline spec/runtime_file_reader_spec.rb]: @arturopie The changes above are just renaming. Only one test was added below.

File: lib/gorgon/runtime_file_reader.rb [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 class RuntimeFileReader
 
-  def initialize(runtime_filename)
-    @runtime_filename = runtime_filename || ""
+  def initialize(configuration)
+    @runtime_filename = configuration[:runtime_filename] || ""
```
Comment (arturopie) [inline lib/gorgon/runtime_file_reader.rb]: You probably want to use ```configuration[:runtime_file]``` since that's what ```RuntimeRecorder``` is using.

**Comment by arturopie** [comment]
@JonErikDSuero, this is awesome. Just fix the bug I pointed out and it will be ready to merge.
Thanks,
--Arturo

**Comment by JonErikDSuero** [comment]
@arturopie I fixed the bug. Please code review again. Thank you

**Pull request merged**